File: DataBinding/Section/XML/VB.NET/XML/ViewerForm.vb
'Viewer window for the XML based ActiveReport
Public Class ViewerForm
	Inherits System.Windows.Forms.Form
	
	Public Sub New()
		MyBase.New()

		'Required designer variable.
		InitializeComponent()
	End Sub

	'Clean up any resources being used.
	Protected Overloads Overrides Sub Dispose(ByVal disposing As Boolean)
		If disposing Then
			If Not (components Is Nothing) Then
				components.Dispose()
			End If
		End If
		MyBase.Dispose(disposing)
	End Sub

	'Set the document to the viewer on the form.
	Public Sub LoadReport(ByVal report As SectionReport)
		arvMain.LoadDocument(report)
	End Sub

End Class 'ViewerForm

File: DataBinding/Section/XML/C#/XML/ViewerForm.cs

namespace GrapeCity.ActiveReports.Samples.XML
{
	/// <summary>
	/// Viewer window for the XML based ActiveReport
	/// </summary>
	public partial class ViewerForm : System.Windows.Forms.Form
	{
		/// <summary>
		///Required designer variable.
		/// </summary>
		public ViewerForm()
		{
			//Required for Windows Form Designer support 
			InitializeComponent();
		}

		/// <summary>
		/// Set the document to the viewer on the form.
		/// </summary>
		public void LoadReport(SectionReport rpt)
		{
			arvMain.LoadDocument(rpt);
		}
	}
}
